app_dart/lib/src/request_handlers/append_log.dart

```diff
@@ -54,6 +66,19 @@ class AppendLog extends ApiRequestHandler<Body> {
       await transaction.commit();
     });
 
+    await writeToStackdriver('${encodedOwnerKey}_${task.attempts}');
+
     return Body.empty;
   }
+
+  /// Write the log data from this request to Stackdriver under [logName].
+  ///
+  /// [logName] must follow the format `[encodedOwnerKey]_[task.attempt]` so each
+  /// attempt for a task can be located.
+  ///
+  /// This will write the log to the global path `projects/flutter-dashboard/logs/[logName]` on Google Cloud.
```

Comment:
Is there a GUI entrance to view these path and logs?

Reply:
The follow up PR to this has more info on the GUI aspect of this change: https://github.com/flutter/cocoon/pull/575